drivers/sensor/sensor_shell.c

```diff
@@ -518,10 +522,11 @@ void sensor_shell_processing_callback(int result, uint8_t *buf, uint32_t buf_len
 					   ch.chan_idx,
 					   data->shift, accumulator_buffer.count,
 					   PRIsensor_q31_data_arg(*data, 0));
-			}
+				}
```

Comment:
Not sure about this indentation change. It makes sense to me the way it was before.